src/components/Scoreboard/Filter.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compareNumbers } from '../../utils';
+import { setNumberOfCardsFilter } from '../../actions/scoreboard';
+
+const NumberOfCardsFilterPure = ({ pastScores, numberOfCards, setNumberOfCardsFilter }) => {
+  const getOptions = (scores) => {
+    const allNumbers = scores.map(score => score.number_of_cards)
+    const distinctNumbers = [ ...new Set(allNumbers) ]
+    distinctNumbers
```

Comment:
Nie możesz robić operacji na `const` jeżeli tak byś chciała to zrobić to musisz zmienić na `let` ;) ale możesz robić to też inaczje, w ogóle ciekawe, że użyłaś `Set` spoko, 
Możesz odrazu zrobić operacje które chcesz czyli :
```
const distinctNumbers = [ ...new Set(allNumbers) ]
.sort(compareNumbers)
.unshift('All')
```

Reply:
poprawione